geoips/plugins/modules/procflows/config_based.py

```diff
@@ -275,8 +296,12 @@ def get_sectored_read(
     else:
         pad_area_def = pad_area_definition(area_def)
     try:
-        xobjs = reader(
-            fnames, metadata_only=False, chans=variables, area_def=pad_area_def
+        xobjs = reader_plugin(
```

Comment:
Renamed to "reader_plugin" for consistency, and easier tracking.  All calls to reader_plugin *should* have **reader_kwargs added to the call.

Reply:
Looks good